gpAux/gpdemo/demo_cluster.sh

```
@@ -362,6 +362,24 @@ echo "gpinitsystem returned: ${RETURN}"
 echo "========================================"
 echo ""
 
+if [ -n "`command -v gpfdist`" ]; then
```

Did give that a thought, but including `src/Makefile.global` means we will need a compiled gpdb_src to create cluster.

I agree with your proposal now, will include `src/Makefile.global` but ignore errors if it's not exists